sdk/src/cameras/itof-camera/camera_itof.cpp

```diff
@@ -83,6 +83,7 @@ CameraItof::CameraItof(
     m_details.uBootVersion = ubootVersion;
     m_details.kernelVersion = kernelVersion;
     m_details.sdCardImageVersion = sdCardImageVersion;
+    m_details.serialNumber = "";
```

Comment:
If serialNumber is of type string, then the default constructor will create an empty string. No need to set it with an empty string.